src/Renderer/WaypointListRenderer.cpp

```diff
@@ -89,7 +89,13 @@ Draw(Canvas &canvas, PixelRect rc,
   row_renderer.DrawSecondRow(canvas, rc, buffer);
 
   // Draw waypoint name
-  row_renderer.DrawFirstRow(canvas, rc, waypoint.name.c_str());
+  if (!waypoint.shortname.empty()) {
+    std::string waypoint_title = waypoint.name+" ("+waypoint.shortname+")";
```

Comment:
This is very inefficient, because it will:
1. copy `waypoint.name`
2. do another copy with `waypoint.name + " ("
3. do another copy of everything so far, appending `waypoint.shortname`
4. do yet another copy of everything so far, appending `")"`
It would be OK to be very inefficient now and optimize later, but this is so bad, it deserves a TODO comment.
But... does this compile on Windows?

Reply:
I understand. The new implementation is now using string append();

Reply:
append() isn't any more efficient than operator+. It's just another name for the same thing.